CONTRIBUTING.md

```diff
@@ -2,9 +2,7 @@
 
 Prometheus uses GitHub to manage reviews of pull requests.
 
-* If you have a trivial fix or improvement, go ahead and create a pull request,
-  addressing (with `@...`) the maintainer of this repository (see
-  [MAINTAINERS.md](MAINTAINERS.md)) in the description of the pull request.
+* If you have a trivial fix or improvement, go ahead and create a pull request.
```

Comment:
I like this part – the extra mention isn't really necessary, so we can remove the instruction :+1: